framework/src/transfers/MultiAppGeneralFieldUserObjectTransfer.C

```diff
@@ -31,6 +31,7 @@ MultiAppGeneralFieldUserObjectTransfer::validParams()
       "Transfers user object spatial evaluations from an origin app onto a variable in the target "
       "application.");
 
+  params.set<std::vector<VariableName>>("source_variable") = std::vector<VariableName>{};
```

Comment:
cant you change the default in generalFieldTransfer instead?

Reply:
"source_variable" is set in 'MultiAppConservativeTransfer' and used multiple places in the code. Most of them take "source_variable" as default parameters, a few not. I think it's better to fix a few places than changing all the rest